trustmanager/pkcs11/common/hardwareprivatekey.go

```diff
@@ -0,0 +1,97 @@
+//+build pkcs11
+
+package common
+
+import (
+	"crypto"
+	"crypto/x509"
+	"fmt"
+	"io"
+
+	"github.com/theupdateframework/notary"
+	"github.com/theupdateframework/notary/tuf/data"
+	"github.com/theupdateframework/notary/tuf/signed"
+)
+
+const (
+	// SigAttempts defines maximum attempts to sign an image before aborting
```

Comment:
Notary is not Docker/container specific, shouldn't reference "image"

Reply:
changed "image" to "artifact"